packages/integrations/node/src/server.ts

```diff
@@ -4,14 +4,14 @@ import createMiddleware from './middleware.js';
 import { createStandaloneHandler } from './standalone.js';
 import startServer from './standalone.js';
 import type { Options } from './types.js';
+applyPolyfills();
```

Comment:
Need to run this first

Reply:
Let's leave a comment so devs know why it's there.